src/WebJobs.Script.WebHost/Diagnostics/Sanitizer.cs

```diff
@@ -12,7 +12,8 @@ namespace Microsoft.Azure.WebJobs.Logging
     internal static class Sanitizer
     {
         private const string SecretReplacement = "[Hidden Credential]";
-        private static readonly char[] ValueTerminators = new char[] { '<', '"' };
+        private static readonly char[] ValueTerminators = new char[] { '<', '"', '\'' };
+        private static readonly string[] PublicTokens = new string[] { "PublicKeyToken=" };
```

Comment:
Let's call this `AllowedTokens` so we can add to it in the future. Add a comment here that these are tokens that we don't want to replace but that match another credential token.